packages/gaudimarlinwrapper/package.py

```diff
@@ -33,5 +35,13 @@ def setup_run_environment(self, spack_env):
         spack_env.prepend_path('PYTHONPATH', self.prefix.python)
         spack_env.prepend_path("PATH", self.prefix.scripts)
 
+    @run_after('install')
+    def install_example(self):
+        install_tree('runit', os.path.join(self.prefix, 'runit'))
+
+    @run_after('install')
+    def install_scripts(self):
+        install_tree('scripts', self.prefix.scripts)
```

Comment:
This one is maybe better to do in CMake. Otherwise the spack and non-spack installations would differ. There is `gaudi_install_scripts()` - I'll prepare a PR in gmp